pressbooks/pressbooks-book-directory-fe
# Remove Featured books section

Related issue: https://github.com/pressbooks/pressbooks-book-directory-fe/issues/181

This change remove completely the Featured section, its components and E2E tests.

### How to test
- Use `dev_pressbooks_directory` index in your environment to make sure the section is anymore displayed since at the moment of this PR there are books with is_featured = true in that index.
- Review the application and you should not see any header section.
